src/test/java/org/apache/maven/plugins/dependency/fromDependencies/TestUnpackDependenciesMojo.java

```diff
@@ -491,16 +491,13 @@ public void dotestClassifierType(String testClassifier, String testType) throws
         }
     }
 
-    public void testArtifactNotFound() throws Exception {
-        dotestArtifactExceptions(false, true);
+    public void testArtifactResolutionException() throws MojoFailureException {
+        dotestArtifactExceptions();
     }
 
-    public void testArtifactResolutionException() throws Exception {
-        dotestArtifactExceptions(true, false);
-    }
-
-    public void dotestArtifactExceptions(boolean are, boolean anfe) throws Exception {
```

Comment:
Removed the parameters from this test method because it looks like they were unused since 809bfe1c4b175e1d684de2da39003d3ecefa1c54, so `testArtifactNotFound` and `testArtifactResolutionException` were effectively equivalent.